firebase-installations-interop/src/main/java/com/google/firebase/installations/FirebaseInstallationsApi.java

```diff
@@ -49,8 +49,8 @@ public interface FirebaseInstallationsApi {
 
   /**
    * Async function that deletes this Firebase app installation from Firebase backend. This call
-   * would possibly lead Firebase Notification, Firebase RemoteConfig, Firebase Predictions or
-   * Firebase In-App Messaging not function properly.
+   * may cause Firebase Cloud Messaging, Firebase Remote Config, Firebase A/B Testing, or
+   * Firebase In-App Messaging to not function properly.
```

Comment:
The `check-changed` CI check is failing due to the formatting here. Autoformat suggested using this instead:
```
   * Async function that deletes this Firebase app installation from Firebase backend. This call may
   * cause Firebase Cloud Messaging, Firebase Remote Config, Firebase A/B Testing, or Firebase
   * In-App Messaging to not function properly.
```

Notice that the word "may" moved from the second line to the first and "Firebase" moved from the third line to the second.

Reply:
Thanks a bunch, that did the trick!